### PR TITLE
[prometheus-node-exporter] Update image to v1.4.0

### PR DESCRIPTION
**Pull request by lukas-unity**
#### What this PR does / why we need it

Node Exporter has released v1.4.0 this PR updates the base image in the chart.  

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-node-exporter/Chart.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.5.2
-appVersion: 1.3.1
```
Comment (monotek) [inline charts/prometheus-node-exporter/Chart.yaml:10]: to use the app version you would need to adjust the deamonset too.
see: https://github.com/prometheus-community/helm-charts/blob/d3bccc205efd0a4dfa7fccc4cda25cf28272c6c6/charts/prometheus-node-exporter/templates/daemonset.yaml#L40

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-node-exporter/Chart.yaml (hunk 1 above)
Comment (lukas-unity) [reply charts/prometheus-node-exporter/Chart.yaml:10]: Thank you, I think I looked at the wrong chart when I changed the `appVersion` 😅

File: charts/prometheus-node-exporter/Chart.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ keywords:
...
+version: 4.5.3
```
[the remark this answers is not in the document]
Comment (lukas-unity) [reply charts/prometheus-node-exporter/Chart.yaml]: Updated

File: charts/prometheus-node-exporter/Chart.yaml (hunk 1 above)
Comment (lukas-unity) [reply charts/prometheus-node-exporter/Chart.yaml:10]: It seems to be already defined here https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/templates/_helpers.tpl#L75-L83

**Review by zanhsieh — APPROVED** [review]

**Review by zanhsieh — APPROVED** [review]

File: charts/prometheus-node-exporter/Chart.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.6.0
-appVersion: 1.3.1
+version: 4.6.1
```
Comment (monotek) [inline charts/prometheus-node-exporter/Chart.yaml:9]: imho chart minor version should be raised, if app minor version has changed

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-node-exporter/Chart.yaml (hunk 3 above)
Comment (lukas-unity) [reply charts/prometheus-node-exporter/Chart.yaml:9]: So you think we should bump it to 4.7.0 ?

Comment (monotek) [reply charts/prometheus-node-exporter/Chart.yaml:9]: yes

Comment (lukas-unity) [reply charts/prometheus-node-exporter/Chart.yaml]: done

**Review by monotek — APPROVED** [review]

**Pull request merged**